src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,61 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import { SearchLink } from './SearchLink';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+const centuriesValue: string[] = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const resetParams = {
+    query: null,
+    sex: null,
+    centuries: null,
+  };
```

Comment:
The `resetParams` object sets `query`, `sex`, and `centuries` to `null`. However, when using `URLSearchParams`, setting a parameter to `null` does not remove it from the URL. Consider using `params.delete('key')` to remove parameters or set them to an empty string if you want to clear them.